### PR TITLE
docs(alpha-features): simplify early access program

### PR DESCRIPTION
**Pull request by HanselIdes**
## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by christinaausley — DISMISSED** [review]

**Review by christinaausley — APPROVED** [review]

**Comment by christinaausley** [comment]
@akeller Could you retroactively review this at your convenience?

File: docs/reference/alpha-features.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Use alpha features to learn about upcoming changes, try them out, and share feedback.
+
+### Alpha
+
+Selected Camunda features and components are released as alpha versions. We release them in an early state for you to test and participate in development by sharing your feedback before they reach general availability (GA).
+
+Characteristics of alpha features and components include:
+
+- Not for production use.
```
Comment (akeller) [inline docs/reference/alpha-features.md:16]: I would make this bold.

**Comment by akeller** [comment]
My only feedback is around alpha features vs. alpha releases. AFAIK, alpha releases are not limited to Enterprise customers. Is the correct?

We talk about two channels in our [release policy]( https://docs.camunda.io/docs/reference/release-policy/) and cluster provisioning - stable and alpha, with alphas stating the following:

> These releases cannot be updated to a newer release, and therefore are not meant to be used in production.

If my cluster is on a stable channel, can I access alpha features? Or do I need to be on an alpha channel to have access to alpha features?

**Comment by HanselIdes** [comment]
@akeller It's not clear

Alpha features in cluster components are available through creating a cluster on an alpha channel.
Alpha features in components outside of the cluster are available by enabling alpha features one by one in the Console.

Should this be added?

**Comment by akeller** [comment]
@HanselIdes yes. I see this distinction about components inside and outside of a cluster as an area of potential confusion.

**Comment by HanselIdes** [comment]
Ok, then they should be changed to reflect that distinction. If you want me to amend the PR, I can get to that later tonight, but it probably won’t make it into this docs release


Eric Lundberg

Senior Product Manager



Schedule a meeting<https://calendly.com/eric-lundberg>

Twitter <https://twitter.com/Camunda>  LinkedIn<https://www.linkedin.com/company/camunda/>



[Camunda Logo]<https://camunda.com/>

________________________________

**Comment by christinaausley** [comment]
@HanselIdes -- @jprcoliveira also has some changes going out tomorrow, so if you want to launch a PR for this, we can merge and push out in a release tomorrow afternoon.

**Pull request merged**